lib/features/children/add_member/cubit/add_member_cubit.dart

```diff
@@ -143,36 +143,49 @@ class AddMemberCubit extends Cubit<AddMemberState> {
 
     emit(state.copyWith(status: AddMemberStateStatus.loading));
     try {
-      await _addMemberRepository.addMembers(members);
-
-      emit(state.copyWith(status: AddMemberStateStatus.success));
-
-      unawaited(
-        AnalyticsHelper.logEvent(
-          eventName: AmplitudeEvents.vpcAccepted,
-        ),
-      );
+      final membersAddedSuccessfully =
+          await _addMemberRepository.addMembers(members);
+
+      if (membersAddedSuccessfully) {
```

Comment:
let's get rid from membersAddedSuccessfully - if we are here then there was no exception